libraries/botbuilder-storage-mssql/README.md

```diff
@@ -0,0 +1,13 @@
+# Bot Builder Microsoft SQL Server Storage
+
+This is a simple storage adapter for Microsoft SQL Server. To use:
```

Comment:
Add config instructions? (+ encrypt instructions for Azure SQL?)

Reply:
I haven't added this yet to the draft. I usually add more Docs after the example and then do the release draft.

Reply:
I found out that this was a draft after my review.. 😂

Reply:
@iMicknl Still good info!

Reply:
Updated README, but I'm unfamiliar with Azure SQL encrypt, so someone else might have to add that.